packages/shared/src/graphql/sources.ts

```diff
@@ -25,6 +25,7 @@ export enum SourcePermissions {
 
 export interface SourceMember {
   role: SourceMemberRole;
+  roleRank: number;
```

Comment:
I think @capJavert removed this number exposure now.
Maybe should align that?

Reply:
Oh, why did we remove it?

Reply:
I removed the one for posting permission (different field on Source). But generally I think frontend does not use numeric ranks anywhere, all checks are done through permissions so maybe this is not needed too.

Reply:
Oh, cool. Now that we have a use case here, we can now probably keep it.

Reply:
@sshanzel but is it no better to evaluate actual permissions like Ante suggested to check if someone can view blocked members? (Seems more inline with what we do)

Reply:
Oh, right, we have a new permission for blocked members. We can probably use that.